docs/security/trust/content_trust.md

```diff
@@ -152,14 +152,36 @@ In an environment where `DOCKER_CONTENT_TRUST` is set, you can use the
 `--disable-content-trust` flag to run individual operations on tagged images
 without content trust on an as-needed basis.
 
-```bash
-$  docker pull --disable-content-trust docker/trusttest:untrusted
+Consider the following Dockerfile that uses an untrusted base image:
+
+```
+$  cat Dockerfile
+FROM docker/trusttest:latest
+RUN echo
+```
+
+In order to build a container successfully using this Dockerfile, one can do:
+
+```
+$  docker build --disable-content-trust -t <username>/nottrusttest:testing .
```

Comment:
Well, I agree that those names make more sense but we have a docker/trusttest:latest image that we were using here as an example and I just thought that it would be good to keep it consistent throughout the document. If we wanna change the names here in the doc, we might even want to add a corresponding image to the registry. I am fine with both. Can do whatever works the best